pkg/observability/modules/suseObservability.ts

```diff
@@ -49,22 +49,22 @@ export function mapKind(kind: string): string {
   return KINDS.get(kind) || kind;
 }
 
-export async function loadStackStateSettings(store: any) {
+export async function loadSuseObservabilitySettings(store: any) {
   const settings = await store.dispatch('management/findAll', { type: OBSERVABILITY_CONFIGURATION_TYPE });
 
   if (!settings || isEmpty(settings)) {
     return;
   }
 
-  const stackstateSettings = settings.find(
-    (s: any) => s.metadata.name === 'stackstate'
+  const suseObservabilitySettings = settings.find(
+    (s: any) => s.metadata.name === 'stackstate' || s.metadata.name === 'suse-observability'
   );
 
-  if (!stackstateSettings || isEmpty(stackstateSettings)) {
+  if (!suseObservabilitySettings || isEmpty(suseObservabilitySettings)) {
```

Comment:
if `suseObservabilitySettings` present then isEmpty(`suseObservabilitySettings`) always returns false

it can just return `settings.find` result straight away